### PR TITLE
New Offers Docs

### PR DESCRIPTION
**Pull request by AlexLakatos**
Deploy preview: https://601ad8ae7a992800ffedd77e--fidel-website.netlify.app/docs/offers

### CODE REVIEW

File: offers/README.md [hunk 1]
```diff
@@ -1,234 +1,295 @@
...
+
+The Offer has at least one Location linked, but the `startDate` is in the future. Offers in this category will automatically become "live" when the `startDate` is reached.
+
+### Live
+
+The current date is between the Offer `startDate` and `endDate`. The Offers in this category are qualifying Transactions at the locations linked to them.
+
+### Expired
+
+The current date is after the Offer `endDate`. The Offers in this category are have stopped qualifying Transactions.
```
Comment (diogogvhenriques) [inline offers/README.md:286]: `The Offers in this category have stopped qualifying Transactions`

File: offers/README.md [hunk 2]
```diff
@@ -1,234 +1,295 @@
...
 
-## Offer object
+## Offer Object
+The Offers API's central piece of data is the Fidel Offer object, which holds all the details about a card-linked Offer. The card-linked offer has a set of parameters used to qualify any [card transaction](https://fidel.uk/docs/transactions) made at a participating Brands' linked Location.
 
-Once you have created an offer, you can request the contents of the object using the [Get Offer](https://reference.fidel.uk/reference#get-offer) API call. The response will appear similar to the following:
+The Offer object looks similar to the following and will be returned by the API in a multitude of situations. Transaction objects will also have a smaller version of the object inside, making it easier to retrieve the full Offer object if necessary.
```
Comment (joao8lopes) [inline offers/README.md:8]: is it worth saying `Qualified transaction objects will(...)`

Comment (AlexLakatos) [reply offers/README.md:8]: From my understanding, all transaction objects will have the offer property. The ones that didn't qualify will have a rejection message, and the qualified ones will have a performanceFee property.

Comment (diogogvhenriques) [reply offers/README.md:8]: Correct, so the offer will appear in the transactions if the transaction is from a location linked to an offer. If the transaction is done on a location that isn't linked to an offer, the transaction.offer = null. But I think the way @AlexLakatos described it's good.

Comment (joao8lopes) [reply offers/README.md:8]: fine for me, sounds good

**Review by diogogvhenriques — COMMENTED** [review]
Reading this with the preview helps a lot 🙏 
The flow of the docs looks great too me, there's some things that I think could be mentioned like Qualified transactions by offers with return period will only qualify after the return period, I think we had already some questions about  it that we could have here mentioned too.

Maybe since you're updating the docs here would be work mention that link all locations from a brand/program https://fidel.atlassian.net/browse/OFF-545, unless you prefer do it afterwards :)

**Comment by miguelmartinzz** [comment]
This looks great @AlexLakatos, awesome job 👏

**Review by diogogvhenriques — APPROVED** [review]

**Pull request merged**